### PR TITLE
avoid boolean coercion

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by stefanpenner** [comment]
restarting

**Comment by locks** [comment]
@stefanpenner it's green

File: packages/ember-metal/lib/chains.js [hunk 1]
```diff
@@ -113,13 +113,11 @@ function addChainWatcher(obj, keyName, node) {
...
 function removeChainWatcher(obj, keyName, node, _meta) {
-  if (!isObject(obj)) {
-    return;
-  }
+  if (!isObject(obj)) { return; }
 
   let meta = _meta || peekMeta(obj);
 
-  if (!meta || !meta.readableChainWatchers()) {
+  if (meta === undefined || meta.readableChainWatchers() === undefined) {
```
Comment (krisselden) [inline packages/ember-metal/lib/chains.js]: there is falsy coercion above this in `let meta = _meta || peekMeta(obj);`

**Review by krisselden — CHANGES_REQUESTED** [review]
Other than the one issue, looks good.

**Comment by krisselden** [comment]
it's spelled coercion in case you want to update the commit messages

**Comment by rwjblue** [comment]
Thanks @bekzod !

**Pull request merged**